### PR TITLE
Implement mobile icon for messages

### PR DESCRIPTION
**Pull request by markwylde**
### Summary
This adds an icon to the mobile view for messages.

image

##### Component Name
N/A

##### Test Plan
Login and reduce your screen to mobile width.

##### Additional Information
- It's possible you may want to change the colours, as the red and white are static HEX and plucked out of the air by me!
- The number '2' is just hard coded in, as I'm not sure how to retrieve that yet.

### CODE REVIEW

File: internal/static/css/99-twtxt.css [hunk 1]
```diff
@@ -34,6 +34,21 @@ article hgroup footer {
   width: 0px !important;
 }
 
+.mobile-menu a {
```
Comment (prologic) [inline internal/static/css/99-twtxt.css:37]: Is this one used anywhere?￼

**Review by prologic — APPROVED** [review]
Very nice I think this will work nicely!

Can you change the pull request to target the ￼`messages_poc_2`?

File: internal/static/css/99-twtxt.css (hunk 1 above)
Comment (markwylde) [reply internal/static/css/99-twtxt.css:37]: Yeah it's the burger menu (and now the messages menu too) anchor.

File: README.md [hunk 2]
```diff
@@ -148,10 +148,10 @@ Then visit: http://localhost:8000/
 Run twtd:
 
 ```console
-twtd -r
+twtd -R
```
[the remark this answers is not in the document]
Comment (markwylde) [reply README.md:151]: :D

**Comment by markwylde** [comment]
> Very nice I think this will work nicely!
> 
> Can you change the pull request to target the ￼`messages_poc_2`?

I think it already is, no?

**Comment by prologic** [comment]
Oh you're absolutely right! How blind of me! 🤣￼

**Pull request merged**